code/dijkstra-algorithm/src/test/java/ru/unn/agile/DijkstraAlgorithm/Model/DijkstraGraphTest.java

```diff
@@ -0,0 +1,171 @@
+package ru.unn.agile.DijkstraAlgorithm.Model;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class DijkstraGraphTest {
+
+    private static final List<DijkstraGraph.Edge> EMPTY_EDGES = Collections.emptyList();
+
+    private static final List<DijkstraGraph.Edge> EDGE_AB =
+            Collections.singletonList(new DijkstraGraph.Edge("a", "b", 7));
+    private static final String EDGE_AB_A_TO_B_PATH = "a(0) -> b(7)";
+    private static final String EDGE_AB_A_TO_A_PATH = "a(0)";
+
+    private static final List<DijkstraGraph.Edge> EDGES_ABCD = Arrays.asList(
+            new DijkstraGraph.Edge("a", "b", 4),
+            new DijkstraGraph.Edge("с", "d", 5)
+    );
+    private static final String CAN_NOT_REACH_D_VERTEX = "Can not reach d vertex";
+
+    private static final List<DijkstraGraph.Edge> EDGES_1 = Arrays.asList(
+            new DijkstraGraph.Edge("a", "b", 4),
+            new DijkstraGraph.Edge("a", "c", 5),
+            new DijkstraGraph.Edge("a", "d", 6),
+            new DijkstraGraph.Edge("b", "e", 8),
+            new DijkstraGraph.Edge("c", "e", 7),
+            new DijkstraGraph.Edge("d", "e", 4)
+    );
+    private static final String EDGES_1_A_TO_E_PATH = "a(0) -> d(6) -> e(10)";
+
+    private static final List<DijkstraGraph.Edge> EDGES_2 = Arrays.asList(
+            new DijkstraGraph.Edge("a", "b", 4),
+            new DijkstraGraph.Edge("a", "c", 5),
+            new DijkstraGraph.Edge("a", "d", 6),
+            new DijkstraGraph.Edge("b", "c", 2),
+            new DijkstraGraph.Edge("d", "c", 1),
+            new DijkstraGraph.Edge("c", "e", 3),
+            new DijkstraGraph.Edge("e", "f", 11),
+            new DijkstraGraph.Edge("e", "g", 5),
+            new DijkstraGraph.Edge("e", "h", 6),
+            new DijkstraGraph.Edge("f", "k", 1),
+            new DijkstraGraph.Edge("g", "k", 7),
+            new DijkstraGraph.Edge("h", "k", 10)
+    );
+    private static final String EDGES_2_A_TO_K_PATH = "a(0) -> c(5) -> e(8) -> g(13) -> k(20)";
+
+    private static final List<DijkstraGraph.Edge> EDGE_AB_NEGATIVE =
+            Collections.singletonList(new DijkstraGraph.Edge("a", "b", -7));
+
+    @Test
+    public void canBuildGraph() {
```

Comment:
Please clarify what is the goal of using this test?
Please follow the rule of AAA.

Reply:
The test was deleted as redundant